roles/manage-users/tasks/add_user_to_group.yml

```diff
@@ -5,17 +5,20 @@
 
```

Comment:
Since you're changing the format of the list of users, you should be able to change the `set_fact` above to read:
```
set_fact:
  user_list: "{{ this_group.members | join(',') }}"
```
... then remove lines 6-15 below.